src/main/java/uk/nhs/hee/tis/trainee/ndw/config/ApplicationConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2023 Crown Copyright (Health Education England)
```

Comment:
Wondering if the copyright is still Health Education England?

Reply:
Probably not, but I think this is preferred until we change the copyright hold across the whole project.